.github/workflows/ci.yaml

```diff
@@ -0,0 +1,65 @@
+---
+name: Generator tests
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  unit-tests:
+    runs-on: ubuntu-latest
+    container: golang:1.13
+    steps:
+      - uses: actions/checkout@v2
+      - run: go test -p 1 ./...
+  bazel-build:
+    runs-on: ubuntu-latest
+    container: gcr.io/gapic-images/googleapis-bazel:20210105
+    # Dockerfile for this image: https://github.com/googleapis/googleapis-discovery/blob/master/Dockerfile
+    # If you update its version, please also update it below in
+    # 'Cache Bazel files' - unfortunately it cannot accept variables at this
+    # time.
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Cache Bazel files
+      id: cache-bazel
+      uses: actions/cache@v2
+      with:
+        path: ~/.cache/bazel
+        key: ${{ runner.os }}-bazel-20210105-${{ secrets.CACHE_VERSION }}
+
+    - name: Cache not found
+      if: steps.cache-bazel.outputs.cache-hit != 'true'
+      run: |
+        echo "No cache found."
+
+    - name: Cache found
+      if: steps.cache-bazel.outputs.cache-hit == 'true'
+      run: |
+        echo -n "Cache found. Cache size: "
+        du -sh ~/.cache/bazel
+        echo "If the cache seems broken, update the CACHE_VERSION secret in"
+        echo "https://github.com/googleapis/googleapis-discovery/settings/secrets/actions"
+        echo "(use any random string, any GUID will work)"
+        echo "and it will start over with a clean cache."
+        echo "The old one will disappear after 7 days."
+
+    - name: Run bazel build
+      run: bazel build '//...'
```

Comment:
Q: This is running `bazel build` at the top of the Showcase repo? If so, where is it specified what version of Go it's using? (the `unit-tests` job has a `golang` container, but I'm not seeing a Go version specified for this job's container (https://github.com/googleapis/googleapis-discovery/blob/master/Dockerfile)

Reply:
I'm not not following, this isn't the Showcase repo, its the Go micro-generator.

The bazel build is using the version of Go specified in the WORKSPACE here https://github.com/googleapis/gapic-generator-go/blob/eadb3631a0520ea9b439d8d7ec4885122589e003/WORKSPACE#L30.

That one is on 1.15.6 because of requirements in the version of rules_go we are using. We need that version of rules_go because of changes in the protobuf and google.golang.org/api repos that we need. The bazel deps are a little bit of a mess tbh.

Reply:
Ah, OK. 
- Sorry, I should have said generator-go; my mistake.
- OK, the WORKSPACE file pins the version. sgtm